index.html

```diff
@@ -3697,6 +3721,10 @@ <h3>Parsing content</h3>
       <li>
         Set |record|'s <a>id</a> to |ndef|'s |id:string|.
       </li>
+      <li>
+        Set |record|'s <a>isLocal</a> to `true` if |ndef| is a payload
+        to another <a>NDEF record</a>, or otherwise `false`.
+      </li>
```

Comment:
Just note to myself, I think this needs one more check. No action for now, I need to think it through.